index.html

```diff
@@ -1,110 +1,326 @@
+        margin: 0;
+      }
+
+      .either fieldset:first-child {
+        border-bottom: 0.5rem solid var(--bg-accent);
+      }
 
-    button {
+      .either input {
+        line-height: 1.5;
+        margin-bottom: 0.5em;
+        width: 100%;
+      }
+
+      button {
         display: block;
         border: none;
         background: var(--bg-accent);
         color: #fff;
         border-radius: 2px;
         padding: 1em;
-    }
+      }
 
-    button:hover {
+      button:hover {
         background: var(--bg-accent-alt);
         cursor: pointer;
-    }
+      }
+
+      code {
+        font-size: 1rem;
+      }
+
+      dt {
+        font-weight: bold;
+      }
+
+      dd {
+        margin-bottom: 1em;
+        margin-top: 1em;
+      }
+
+      summary h2 {
+        display: inline;
+        line-height: 3em;
+      }
 
-    footer {
+      footer {
+        margin: 3em 0;
+        border-top: 2px solid;
+        padding: 0.5em 0;
         text-align: center;
-        margin: 1em;
-        margin-top: 5em;
-        border-top: 0.2rem solid;
-        padding: 0.5em;
-    }
+      }
+
+      body:has(#type-respec:checked) .only-bikeshed-spec,
+      body:has(#type-respec:checked) .only-bikeshed-issues-list,
+      body:has(#type-respec:checked) .non-respec,
+      body:has(#type-bikeshed-spec:checked) .only-respec,
+      body:has(#type-bikeshed-spec:checked) .only-bikeshed-issues-list,
+      body:has(#type-bikeshed-issues-list:checked) .only-respec,
+      body:has(#type-bikeshed-issues-list:checked) .only-bikeshed-spec,
+      body:has(#type-bikeshed-issues-list:checked) .non-bikeshed-issues-list {
+        display: none;
+      }
     </style>
-</head>
-<body>
+  </head>
+
+  <body>
     <form action="">
-        <h2>Convert ReSpec document to HTML</h2>
-        <fieldset style="text-align: right;">
-            <label for="type">Generator</label>
-            <select name="type" id="type" required>
-                <option value="respec" selected>ReSpec</option>
-            </select>
-        </fieldset>
-        <div class="either">
-            <fieldset>
-                <label for="url">Spec URL (with optional query parameters)</label>
-                <input type="url" name="url" id="url" placeholder="https://w3c.github.io/manifest/?specStatus=WD&publishDate=2020-10-20">
-                <button formmethod="GET">Generate</button>
-            </fieldset>
-            <fieldset>
-                <label for="file">Or, upload a HTML or tar file</label>
-                <input type="file" name="file" id="file" accept="application/x-tar, .html">
-                <button formmethod="POST" formenctype="multipart/form-data">Upload and Generate</button>
-            </fieldset>
+      <h1>Convert document to HTML</h1>
+      <fieldset>
+        <legend>Document type</legend>
+        <input
+          id="type-respec"
+          type="radio"
+          name="type"
+          value="respec"
+          checked
```

Comment:
Small enhancement: If url parameter has `?type` set, we can check the right radio buttons with script.